cloudify_ssh_key/operations.py

```diff
@@ -192,10 +192,14 @@ def _create_secret(key, value):
 def _get_secret(key):
     try:
         client = manager.get_rest_client()
-        return client.secrets.get(key)
+        response = client.secrets.get(key)
     except CloudifyClientError as e:
         raise NonRecoverableError(str(e))
 
+    if not response.value:
+        raise NonRecoverableError('')
+    return client.secrets.get(key)
```

Comment:
An error message would be nice , @EarthmanT what do you think ?
the other thing , why call the get secret api again [ extra overhead ]
```suggestion
    if not response.value:
        raise NonRecoverableError('{0} secret is empty'.format(key))
    return response
```